plugins/basic/nexus-ui-extjs3-plugin/src/main/resources/static/js/repoServer/RepoMaintPanel.js

```diff
@@ -534,6 +534,17 @@ Ext.extend(Sonatype.repoServer.RepositoryPanel, Sonatype.panels.GridViewer, {
               }
             }
           }
+          else
+          {
+            this.dataStore.each(function(rec) {
+              if (rec.get('repoType') !== 'group') {
+                rec.beginEdit();
+                rec.set('displayStatus', '');
```

Comment:
This clears the 'retrieving...' display status when we send back a response with no content.